airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -1666,6 +1684,8 @@ def __init__(
         project_id: Optional[str] = None,
         location: Optional[str] = None,
         job_id: Optional[str] = None,
+        force_rerun: bool = False,
```

Comment:
Hmm these defaults don't seem to make sense to to me.
Why is `force_rerun = False` but `reattach_states = None` (eventually empty set) which means by default if there is an existing job id reattach candidate the user will be sad with this airflow exception telling them "should've set force_rerun to true.
We should either default force_rerun=True or have a default set of reattach states.
IMO user expectation is force_rerun=True.

Reply:
> IMO user expectation is force_rerun=True

I agree, that's also the old behavior